### ISSUE TITLE
custom fields should be exported as-is

### ISSUE THREAD

**Issue by pprw** [ZotPlus/zotero-better-bibtex#441]
I would like to insert into Zotero some Biblatex fields. To be specific the "related" and "relatedtype" fields which are useful to print a reference with its translation (among other things).

I followed [the documentation](https://zotplus.github.io/zotero-better-bibtex/export.html) which says:

> You can add any field you like by using something like bibtex[origdate=1856;origtitle=An Old Title] in the extra field of your reference.

And I added into the "extra" field of my entry this code:

```
bibtex[related=key_2;relatedtype=bytranslator]

```

The problem is that BBT translates this (when exporting to biblatex) into this:

```
...
  related = {key\_2},
  relatedtype = {bytranslator}
...
```

You can notice the "\" added which will prevent Biblatex to understand this field. I tried `bibtex[related="key_2";relatedtype=bytranslator]` without success. It seems BBT cannot export the `_`. BBT converts it to `\_`.

Here is a LaTeX example of want I want
```
 \begin{filecontents*}{biblio.bib}
 @article{key_1,
   author =   {Author},
   title =    {Title},
   journaltitle = {Journal},
   year =     2015,
   number = 2,
   volume = 167,
   pages = {67--88},

   related = {key_2},
   relatedtype = {bytranslator}
 }

 @article{key_2,
   author =   {Author},
   title =    {Title translated},
   journaltitle = {Other Journal},
   year =     2015,
   number = 1,
   volume = 168,
   pages = {88--98},
   translator = {Translator}}

 \end{filecontents*}

\documentclass[12pt,twoside,a4paper]{article}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage{filecontents}
\usepackage[style=authoryear-comp,backend=biber]{biblatex}
\addbibresource{biblio.bib}
\begin{document}

\cite{key_1}


\printbibliography
\end{document}
```
Result: ![screenshot](https://cloud.githubusercontent.com/assets/5262986/12651927/0516d064-c5e9-11e5-82aa-02073a8bd8cd.png)

An I have this with BBT (which will not work because of the *\*):

```
@article{author_title_2015-1,
  title = {Title {{Translated}}},
  volume = {168},
  timestamp = {2016-01-28T16:44:29Z},
  number = {1},
  journaltitle = {Other Journal},
  shortjournal = {J.},
  author = {{Author}},
  translator = {{Translator}},
  date = {2015},
  pages = {88--98}
}

@article{author_title_2015,
  title = {Title},
  volume = {167},
  timestamp = {2016-01-28T16:44:29Z},
  number = {2},
  journaltitle = {Journal},
  shortjournal = {ET J.},
  author = {{Author}},
  date = {2015},
  pages = {67--88},
  related = {key\_2},
  relatedtype = {bytranslator}
}
```

**Comment by retorquere** [comment]
Found the problem, tests are running.

**Comment by retorquere** [comment]
Fixed in 1.6.34, thanks for reporting this.

**Comment by pprw** [comment]
You're welcome. Thanks for the bug fix.
